### DISCUSSION TITLE
visual and caret mode navigation

### DISCUSSION THREAD

**Discussion by Craigdreams** [gdh1995/vimium-c#1030] [Q&A]
Hi, what are all possible visual and caret mode navigation commands?  these are some of the ones iv found looking through the code:
   "l": "forward character",
  "h": "backward character",
  "j": "forward line",
  "k": "backward line",
  "e": "forward word",
  "b": "backward word",
  "w": "forward vimword",
  ")": "forward sentence",
  "(": "backward sentence",
  "}": "forward paragraph",
  "{": "backward paragraph",
  "0": "backward lineboundary",
  "$": "forward lineboundary",
  "G": "forward documentboundary",
  "gg": "backward documentboundary",

I believe this should also be added to the help menu.

**Comment by gdh1995** [comment]
You may find all built-in VisualMode key mappings in: https://github.com/gdh1995/vimium-c/blob/492caa7ac83df864da8f3058573dd67b524f0a6f/background/key_mappings.ts#L855-L881
